crates/uv-python/src/virtualenv.rs

```diff
@@ -86,6 +86,10 @@ impl CondaEnvironmentKind {
     /// `/usr/local/conda/envs/<name>`. Note the name `CONDA_DEFAULT_ENV` is misleading, it's the
     /// active environment name, not a constant base environment name.
     fn from_prefix_path(path: &Path) -> Self {
+        if is_pixi_environment(path) {
```

Comment:
We should probably have a comment here? Like 
```suggestion
        // Pixi never creates "base" environments but uses the name "default" for project environments
        // which breaks our heuristics, so we special case them
        if is_pixi_environment(path) {
```